cibuildwheel/resources/cibuildwheel.schema.json

```diff
@@ -890,6 +890,11 @@
         },
         "test-requires": {
           "$ref": "#/properties/test-requires"
+        },
+        "pyodide-version": {
```

Comment:
This file gets generated, I don't see the source being updated?

Reply:
🤔 Just tried `nox -s generate_schema` and the changes went away. I'll check how to update the schema properly.

Reply:
Doesn't cibuildwheel/options.py also need updating? I don't see a `.get("pyodide-versions")` being read there.